tests/config/test_data.config

```diff
@@ -24,5 +25,11 @@ params {
                 b0_mask = "${params.test_data_base}/main/.test_data/preproc/b0_mask.nii.gz"
             }
         }
+        "denoising" {
+            "nlmeans"{
+                image = "${params.test_data_base_denoise}/main/.test_data/denoising/nlmeans_image.nii.gz"
```

Comment:
params.test_data_base_denoise -> params.test_data_base

here and the line below